consensus/progpow/consensus.go

```diff
@@ -713,3 +715,28 @@ func (progpow *Progpow) FinalizeAndAssemble(chain consensus.ChainHeaderReader, h
 func (progpow *Progpow) NodeLocation() common.Location {
 	return progpow.config.NodeLocation
 }
+
+// AccumulateRewards credits the coinbase of the given block with the mining
+// reward. The total reward consists of the static block reward and rewards for
+// included uncles. The coinbase of each uncle block is also rewarded.
+func (progpow *Progpow) accumulateRewards(config *params.ChainConfig, state *state.StateDB, header *types.WorkObject, logger *log.Logger) {
+	// Select the correct block reward based on chain progression
+	blockReward := misc.CalculateReward(header, progpow.DiffToBigBits(header))
+
+	coinbase, err := header.Coinbase().InternalAddress()
+	if err != nil {
+		logger.WithFields(log.Fields{
+			"Address": header.Coinbase().String(),
+			"Hash":    header.Hash().String(),
+		}).Error("Block has out of scope coinbase, skipping block reward")
```

Comment:
Do we want to still consider this block as valid? Or else return an error here and pick it up in the caller.

Reply:
Yes. We have had this conversation previously and decided to keep these blocks.